src/Octoshift/Services/GithubApi.cs

```diff
@@ -225,6 +225,41 @@ public virtual async Task<string> GetOrganizationId(string org)
         }
     }
 
+    public virtual async Task<string> GetOrganizationIdWithDatabaseId(string org)
+    {
+        var url = $"{_apiUrl}/graphql";
+
+        var payload = new
+        {
+            query = @"
+            query($login: String!) {
+                organization(login: $login) {
+                    login
+                    id
+                    name
+                    databaseId
+                }
+            }",
+            variables = new { login = org }
+        };
+
+        try
+        {
+            return await _retryPolicy.Retry(async () =>
+            {
+                var data = await _client.PostGraphQLAsync(url, payload);
+
+                var databaseId = data["data"]["organization"]["databaseId"].ToString();
```

Comment:
We need to cast to a string rather than calling `ToString()` here because if in case of a `null` value `ToString()`  will through be a cast to a `string` won't and since we are not using nullable reference types the method can also return `null`:

```suggestion
                var databaseId = (string)data["data"]["organization"]["databaseId"];
```